csharp/ql/src/change-notes/2024-09-18-csharp-log-forging-enum.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
```

Comment:
```suggestion
category: minorAnalysis
```

Reply:
This is as such not a "fix" but rather a change in, how we perceive enums.

Reply:
That makes sense.